### DISCUSSION TITLE
transcript generation

### DISCUSSION THREAD

**Discussion by actsasgeek** [ManimCommunity/manim-voiceover#32] [Ideas]
It would be very cool if this could also generate a timestamped transcript. I have students with various disabilities that benefit from having a transcript.

**Comment by 100Khushi** [comment]
Absolutely! 
Adding a timestamped transcript feature would indeed enhance accessibility. While I can't directly implement this, you might want to explore Wavel's [transcription](https://wavel.ai/solutions/transcriptions) tool as an alternative. It offers accurate transcriptions with timestamps, potentially addressing the needs of your students with disabilities.
